CIP-1694/README.md

```diff
@@ -192,23 +192,31 @@ The following table details the ratification requirements for each governance ac
   The DRep vote threshold that must be met as a percentage of *active voting stake*, ranging from 0 to 100 (inclusive).
 
 * **AVST**<br/>
-  The _**A**ctive **V**oting **S**take **T**hreshold_. The percentage to be used to determine if there is *sufficient active voting stake*. SPO vote endorsement is not required for the specific governance action if there is a sufficient active voting stake.
-  Otherwise, SPO vote endorsement is required. The :x: symbol indicates that the AVST will be ignored, and only the DRep and SPO votes will be considered regardless of the AVST.
+  The _**A**ctive **V**oting **S**take **T**hreshold_. The percentage to be used to determine if there is *sufficient active voting stake*.
+
+* **AVST action**<br/>
```

Comment:
This section is a bit confusing. First you say it's an _action_, then you say it's the _consequence_ of missing the AVST, then you say one option is "required" which is neither an action nor a consequence. 

How about
```
AVST fallback
The fallback condition if the AVST threshold is not met
* None: there is no fallback, the action cannot be ratified unless the AVST threshold is met
* SPO vote: the action can be ratified if there is a sufficient SPO vote endorsement.
```

Reply:
yep, that is much better, thank you!